features/netsocket/emac-drivers/TARGET_STM_EMAC/stm32xx_emac.cpp

```diff
@@ -243,20 +243,20 @@ bool STM32_EMAC::link_out(emac_mem_buf_t *buf)
  * @return a memory buffer filled with the received packet (including MAC header)
  *         NULL on memory error
  */
-emac_mem_buf_t *STM32_EMAC::low_level_input()
+
+int8_t STM32_EMAC::low_level_input(emac_mem_buf_t **buf)
```

Comment:
Doxygen comment not been updated for new prototype.

I wouldn't mess with int8_t returns or parameters on something we know is ARM-based - `int` is faster, matching register size. (Save narrow types for structures)

Reply:
Updated pull request.